style.css

```diff
@@ -0,0 +1,231 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+body {
+  font-family: 'Poppins';
+}
+
+header > div {
+    padding: 0 25px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid lightgray;
+  }
+  
+  nav {
+    width: 600px;
+  }
+  
+  nav ul {
+    list-style: none;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  
+  nav a {
+    text-decoration: none;
+    color: black;
+  }
+  
+  .blackHeart {
+    width: 20px;
+    margin-right: 1rem;
+  }
+
+  .npm-logo {
+    width: 70px;
+    height: 22px;
+    margin-right: 0.75rem;
+    cursor: pointer;
+  }
+  
+  .magnifying-glass {
+    width: 15px;
+    height: 15px;
+    background-color: rgba(0,0,0,.05);
+    cursor: pointer;
+  }
+
+  .magnifying-glass-span {
+    box-sizing: border-box;
+    width: 35px;
+    height: 40px;
+    padding: 14px 4px 0px 16px;
+    background-color: rgba(0,0,0,.05);
+  }
+
+  .search-span {
+    padding: 8px 0px;
+  }
+
+  #search-bar {
+    width: 600px;
+    height: 40px;
+    background-color: rgba(0,0,0,.05);
+    border: none;
+    cursor: pointer;
+  }
+
+  #search-bar-btn {
+    width: 83px;
+    height: 40px;
+    padding: 8px 16px;
+    color: white;
+    background-color: #FB3E44;
+    border: none;
+    cursor: pointer;
+  }
+
+  #join-btn {
+    text-decoration: none;
+    background-color: #fff;
+    font-size: 14px;
+    color: #4b4949;
+    margin: 0px 16px 0px 0px;
+    box-sizing: border box;
+    padding: 4px 16px;
+    border: solid #adaaaa 1px
+  }
+
+  #login-btn {
+    text-decoration: none;
+    background-color: #fff;
+    font-size: 14px;
+    border: none;
+    color: #4b4949;
+  }
+
+  .top-section {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    background-color: rgba(232,217,217,.3);
+    height: 600px;
+  }
+
+  #section-h1 {
+    box-sizing: border-box;
+    font-size: 5rem;
+    margin: 0.60em 0;
```

Comment:
If you want to manipulate the top margin only, I would delete this part. 
If you want to manipulate all of them, I would delete the "margin-top" from below and use this to do what you need.